### ISSUE TITLE
vcf2db error messages

### ISSUE THREAD

**Issue by matthdsm** [chapmanb/bcbio-nextgen#1942]
Hi Brad,

When creating a db vcfanno reports the following:
```
[2017-05-22T07:21Z] skipping 'af_adj_exac_afr' because it has Number=A
[2017-05-22T07:21Z] skipping 'af_adj_exac_amr' because it has Number=A
[2017-05-22T07:21Z] skipping 'af_adj_exac_eas' because it has Number=A
[2017-05-22T07:21Z] skipping 'af_adj_exac_fin' because it has Number=A
[2017-05-22T07:21Z] skipping 'af_adj_exac_nfe' because it has Number=A
[2017-05-22T07:21Z] skipping 'af_adj_exac_oth' because it has Number=A
[2017-05-22T07:21Z] skipping 'af_adj_exac_sas' because it has Number=A
[2017-05-22T07:21Z] skipping 'af_exac_all' because it has Number=A
[2017-05-22T07:21Z] skipping 'max_aaf_all' because it has Number=A
```
I thought this was already fixed in the past, but it seems to be back. Could those values be parsed post vcfanno, so the columns get added to the db?

Thanks
M

**Comment by matthdsm** [comment]
so actually [this](https://github.com/chapmanb/bcbio-nextgen/commit/f27f26c945ba415a55b2b8816af651f75f9a8b5f) should be executed after the vcfanno annotation, or on both occasions, but I'll leave that up to you, since I'm not entirely sure what the effect will be.

Cheers
M

**Comment by matthdsm** [comment]
fixed in #1941
